go/samples/coffee-shop/main.go

```diff
@@ -104,7 +104,7 @@ func main() {
 		fmt.Fprintln(os.Stderr, "You can get an API key at https://ai.google.dev.")
 		os.Exit(1)
 	}
-	err := googleai.Init(context.Background(), googleai.Config{APIKey: apiKey})
+	_, _, err := googleai.Init(context.Background(), googleai.Config{APIKey: apiKey})
```

Comment:
Just to understand, we could do instead here:

gs, _, err := googleai.Init(context.Background(), googleai.Config{APIKey: apiKey, Models: []string{"gemini-1.5-pro"}})
g := gs[0]

?

It seems weird to me that the empty list means "initialize them all". In the call above, the plugin will initialize all the embedders because we didn't specify, and didn't need, any embedders. Maybe there is a more explicit way to say "all"? config.Models == []string{"*"}?
(Fine to tackle in a separate CL.)